ddp-workspace/projects/ddp-dsm-ui/src/app/participant-page/participant-page.component.ts

```diff
@@ -1629,6 +1629,8 @@ export class ParticipantPageComponent implements OnInit, OnDestroy, AfterViewChe
                 canBeSequencedBasedOnLocation = true;
               }
             }
+          } else if(countryQuestion.answer.indexOf(this.SELF_COUNTRY_CA) > -1) {
```

Comment:
**_#optional_**
`Array.prototype.includes()` can be used here to simplify. It returns boolean value.